package.json

```diff
@@ -0,0 +1,138 @@
+{
+  "name": "@rsksmart/rif-storage-upload-service",
+  "version": "0.0.1",
+  "description": "Service for Uploading files to Storage Providers(IPFS, Swarm, ...)",
+  "keywords": [
+    "RIF",
+    "RSK"
+  ],
+  "homepage": "https://github.com/rsksmart/rif-storage-upload-service",
+  "bugs": {
+    "url": "https://github.com/rsksmart/rif-storage-upload-service/issues/"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Nazar Duchak",
+    "email": "nazar@iovlabs.org"
+  },
+  "files": [
+    "/oclif.manifest.json",
+    "/lib",
+    "/config/default.json5",
+    "/config/custom-environment-variables.json5",
+    "/bin"
+  ],
+  "directories": {
+    "lib": "src",
+    "test": "test/",
+    "config": "config/"
+  },
+  "main": "lib/index.js",
+  "bin": {
+    "rif-storage-upload-service": "./bin/run"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rsksmart/rif-storage-upload-service.git"
+  },
+  "scripts": {
+    "prepack": "tasegir run --watch node_modules/.bin/oclif-dev -- manifest && sed -i '' 's#\"./src/cli\"#\"./lib/cli\"#g' package.json",
+    "postpack": "sed -i '' 's#\"./lib/cli\"#\"./src/cli\"#g' package.json",
+    "bin": "tasegir run ./bin/run -- ",
+    "init": "./init.sh",
+    "compile": "tasegir compile",
+    "ipfs": "cross-env IPFS_PATH='./.repos/upload' ipfs",
+    "docs": "typedoc --mode modules --excludeNotExported --readme none --excludePrivate  --tsconfig ./node_modules/tasegir/src/config/tsconfig.json --exclude 'src/providers/*,test/**/*' --out docs src",
+    "types-check": "tasegir types-check",
+    "lint": "tasegir lint",
+    "release": "tasegir release",
+    "test": "npm run test:unit && npm run test:integration",
+    "test:integration": "cross-env ALLOW_CONFIG_MUTATIONS=true tasegir test --target node --files ./test/integration/**/*.spec.ts --files ./test/integration/**/**/*.spec.ts",
+    "test:unit": "cross-env ALLOW_CONFIG_MUTATIONS=true tasegir test --target node --files ./test/unit/*.spec.ts --files ./test/unit/**/*.spec.ts --files ./test/unit/**/**/*.spec.ts",
+    "start": "tasegir run --watch ./src/index.ts",
+    "tasegir:link": "ln -s ./node_modules/tasegir/src/config/tsconfig.json ./tsconfig.json && ln -s ./node_modules/tasegir/src/config/eslintrc.js ./.eslintrc.js"
+  },
+  "dependencies": {
+    "@feathersjs/errors": "^4.5.3",
+    "@feathersjs/express": "^4.5.4",
+    "@feathersjs/feathers": "^4.5.1",
+    "@feathersjs/socketio": "^4.5.4",
+    "@feathersjs/transport-commons": "^4.5.3",
+    "@oclif/command": "^1.8.0",
+    "@oclif/config": "^1.17.0",
+    "@oclif/errors": "^1.3.3",
+    "@oclif/parser": "^3.8.5",
+    "@oclif/plugin-help": "^3.2.0",
+    "@rsksmart/rif-communications-pubsub": "0.1.0-dev.10",
+    "colors": "^1.4.0",
+    "compression": "^1.7.4",
+    "config": "^3.3.1",
+    "cors": "^2.8.5",
+    "feathers-hooks-common": "^5.0.3",
+    "feathers-sequelize": "^6.1.0",
+    "helmet": "^4.0.0",
+    "ipfs-http-client": "^47.0.1",
```

Comment:
Lets use the latest dependencies!

Reply:
dependabot already create a PR with that update